gm4_enchantment_extractors/data/enchantment_extractors/functions/create.mcfunction

```diff
@@ -0,0 +1,5 @@
+summon armor_stand ~ ~-1.2 ~ {Invisible:1b,Invulnerable:1b,Marker:1b,NoGravity:1,Tags:["gm4_enchantment_extractor","gm4_no_edit"],ArmorItems:[{},{},{},{id:enchanting_table,Count:1}],DisabledSlots:2039583}
+data merge block ~ ~ ~ {CustomName:"\"Enchantment Extractor\"",Items:[]}
+advancement grant @a[distance=..3] only gm4:enchantment_extractors
```

Comment:
`@a[distance=..3]` should be `@a[distance=..3,gamemode=!spectator]` to exclude spectators. This prevents them getting advancements whilst spectating people which has been an issue with previous versions of multiple modules.